.github/workflows/ci-cd.yml

```
@@ -0,0 +1,66 @@
+      - name: Build Docker image
+        run: docker build -t nexus:8082/demo-frontend:${{ github.run_number }} .
+
+      - name: Push Docker image
+        run: docker push nexus:8082/demo-frontend:${{ github.run_number }}
+
+  update-manifests:
+    needs: build-and-push-docker
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout manifest repo
+        uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)